packages/front-end/components/Experiment/MetricsSelector.tsx

```diff
@@ -14,17 +15,63 @@ type MetricOption = {
   tags: string[];
   projects: string[];
   factTables: string[];
+  userIdTypes: string[];
 };
 
+export function MetricsSelectorTooltipBody(onlyBinomial: boolean = false) {
```

Comment:
should probably be a JSX component so that you can embed it via `<MetricsSelectorTooltipBody onlyBinomial={true} />`

Reply:
yeah, should do. thanks.